File: 프로그래머스/1/17681. ［1차］ 비밀지도/［1차］ 비밀지도.py
def solution(n, arr1, arr2):
    ans = []
    
    for i in range(n):
        chk = int(arr1[i]) | int(arr2[i])
        lst = bin(chk)[2:].zfill(n)  # 이진수로 변환하고 앞에 0을 채워 길이를 맞춤
        lst = lst.replace("0", " ").replace("1", "#")
        ans.append(lst)
    return ans
    

File: 프로그래머스/1/17681. ［1차］ 비밀지도/［1차］ 비밀지도.js
function solution(n, arr1, arr2) {
    let ans = [];

    for (let i = 0; i < n; i++) {
        let chk = arr1[i] | arr2[i];
        let lst = chk.toString(2).padStart(n, '0'); // 2진수로 변환하고 앞에 0을 채워 길이를 맞춤
        lst = lst.replace(/0/g, ' ').replace(/1/g, '#');
        ans.push(lst);
    }
    
    return ans;
}